dashboard/views/login.ejs

```diff
@@ -1,15 +1,13 @@
 <% include includes/header %>
 <nav class="navbar navbar-transparent navbar-absolute">
     <div class="container-fluid">
-        <div class="collapse navbar-collapse">
-            <div class="navbar-form navbar-right">
-                <select class="form-control" id="languageSelection">
-                  <option value="en">English</option>
-                  <option value="es">Spanish</option>
-                  <option value="fr">French</option>
-                  <option value="hi">Hindi</option>
-                </select>
-            </div>
+        <div class="navbar-form navbar-right pull-right">
+            <select class="form-control" id="languageSelection">
+              <option value="en">English</option>
+              <option value="es">Spanish</option>
```

Comment:
formatting looks little messed. please rectify.

Reply:
Let me work on that.

Reply:
I just made a new commit with the code formatted.